<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pSCT</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Home</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>350</y>
         <width>201</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>Camera Positiion</string>
       </property>
       <widget class="QPushButton" name="pushButton_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>117</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Get position</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>39</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>El/deg</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>44</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Az/deg</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>104</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>105</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>120</y>
         <width>501</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Camera power</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>20</y>
          <width>51</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>471</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Disable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>51</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Voltage</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>230</y>
         <width>511</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Shutter control</string>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Current position</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>471</width>
          <height>44</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>340</y>
         <width>221</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Chiller monitor</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>110</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Pressure</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>420</y>
         <width>221</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Networking monitor</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>PTP packages</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>10</y>
         <width>501</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Fan</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>20</y>
          <width>51</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>51</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Voltage</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>471</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Fans ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Fans OFF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>521</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
